server/channels/store/sqlstore/user_store.go

```diff
@@ -714,11 +714,13 @@ func (us SqlUserStore) GetEtagForProfiles(teamId string) string {
 func (us SqlUserStore) GetProfiles(options *model.UserGetOptions) ([]*model.User, error) {
 	isPostgreSQL := us.DriverName() == model.DatabaseDriverPostgres
 	query := us.usersQuery.
-		Join("TeamMembers tm ON ( tm.UserId = u.Id AND tm.DeleteAt = 0 )").
-		Where("tm.TeamId = ?", options.InTeamId).
```

Comment:
Interesting, we were joining by default?

Reply:
I realized that `tm.DeleteAt` needs to be there always. I'll revert this.